coops/Titans_stronghold.tex

```diff
@@ -0,0 +1,201 @@
+
+\textbf{Win:} Kill all unit in the Titans' Stronghold (the center of tile VI-VII next to the tiles I)
+
+\subsection*{\MakeUppercase{Timed events}}
+
+\textbf{4$^{th}$, 8$^{th}$ and 12$^{th}$ Round:}
+\begin{itemize}
+    \item Remove black cubes from windmills, water wheels and mystical gardens.
+\end{itemize}
+
+\end{multicols*}
+
+\newpage
+
+\subsection*{\MakeUppercase{Additional rules}}
+
+\begin{itemize}
+    \item Remember: the center of tile VI-VII next to all I, is the Titans' Stronghold.
+    \item Nobody can enter the Titans' Stronghold before all other VII field are flagged by any player.
+    \item After defeating a level VII neutral army, instead of resolving field the player chooses an option three times from the following list (an option may be chosen multiple times):
+    \begin{itemize}
+        \item Another player (you choose) gains 5 gold
+        \item Another player (you choose) gains 2 building materials
+        \item Another player (you choose) gains 1 valuable
+    \end{itemize}
+    Then flag the VII with faction cube.
+    (In solo play : no bonus!)
+    \item Ignore all yellow line in tiles I.
+    \item You can use your build token to give your resources to another player.
+    \item Two player can use your build token for exchange artifacts and/or spells.
+    \item Whenever a player visits an Obelisk, that player rolls one tresure die and one resource die, chooses one of the dice and resolves its outcome.
+    \item When all VII (not including the Titans' Stronghold) are flagged, randomly draw and shuffle the following (see the next page) number of Neutral Unit cards from each of their corresponding decks to create a separate deck of Neutral Units for the Titans' Stronghold (the deck of Titans' Stronghold is sometimes cut in two because otherwise certain scenarios are impossible to implement with the cards from certain expansions).
+    \item Any time a Hero enters the Titans' Stronghold they draw 5 cards from the Titans' Stronghold deck instead of drawing them from the Neutral Unit card decks. The units are placed on the Combat board (see page 29, “Neutral Unit Setup” in the Core Rulebook). Players attempt to beat the units they find at the Titans' Stronghold. Any Neutral Units defeated during Combat in the Titans' Stronghold are returned to their respective Neutral Unit decks instead of Titans' Stronghold deck. Any Neutral Units surviving Combat at the Titans' Stronghold are shuffled back into the Titans' Stronghold deck. If there are not enough Unit cards in this deck, draw as many Unit cards as there are and place them on the Combat board.
+    \item Combat in the Titans' Stronghold now costs 1 MP to extend per Combat round, just like Combat against non-Azure tier units.
+    \item Additionally, no player can:
+    \begin{itemize}
+        \item Attack other Heroes.
+        \item Capture a Mine or Settlement that is already controlled.
+    \end{itemize}.
+\end{itemize}
+
+
```

Comment:
```suggestion
\subsection*{\MakeUppercase{Additional Rules}}

\begin{itemize}
    \item Remember: the center of tile VI-VII next to all I tiles is the Titans' Stronghold.
    \item Nobody can enter the Titans' Stronghold before all other VII fields are flagged by any player.
    \item After defeating a level VII neutral army, instead of resolving the field, the player chooses an option three times from the following list (an option may be chosen multiple times):
    \begin{itemize}
        \item Another player (you choose) gains 5 gold.
        \item Another player (you choose) gains 2 building materials.
        \item Another player (you choose) gains 1 valuable.
    \end{itemize}
    Then flag the VII field with a faction cube.
    (In solo play: no bonus!)
    \item Ignore all yellow borders in tiles I.
    \item You can use your build token to give your resources to another player.
    \item Two players can use their build tokens to exchange artifacts and/or spells.
    \item Whenever a player visits an Obelisk, that player rolls one treasure die and one resource die, chooses one of the dice, and resolves its outcome.
    \item When all VII fields (not including the Titans' Stronghold) are flagged, randomly draw and shuffle the following (see the next page) number of Neutral Unit cards from each of their corresponding decks to create a separate deck of Neutral Units for the Titans' Stronghold (the deck of Titans' Stronghold is sometimes cut in two because otherwise certain scenarios are impossible to implement with the cards from certain expansions).
    \item Any time a Hero enters the Titans' Stronghold, they draw 5 cards from the Titans' Stronghold deck instead of drawing them from the Neutral Unit card decks. The units are placed on the Combat board (see page 29, “Neutral Unit Setup” in the Core Rulebook). Players attempt to defeat the units they find at the Titans' Stronghold. Any Neutral Units defeated during Combat in the Titans' Stronghold are returned to their respective Neutral Unit decks instead of the Titans' Stronghold deck. Any Neutral Units surviving Combat at the Titans' Stronghold are shuffled back into the Titans' Stronghold deck. If there are not enough Unit cards in this deck, draw as many Unit cards as there are and place them on the Combat board.
    \item Combat in the Titans' Stronghold now costs 1 MP to extend per Combat round, just like Combat against non-Azure tier units.
    \item Additionally, no player can:
    \begin{itemize}
        \item Attack other Heroes.
        \item Capture a Mine or Settlement that is already controlled.
    \end{itemize}
\end{itemize}
```